components/TestimoniesTable/TestimoniesTable.js

```diff
@@ -1,23 +1,27 @@
 import React from "react"
 import { Table, Container, Button } from "react-bootstrap"
 import ExpandTestimony from "../ExpandTestimony/ExpandTestimony"
+import { useRouter } from "next/router"
 import { useBill, usePublicProfile } from "../db"
 import Link from "next/link"
 import { formatBillId } from "../formatting"
 import ProfileButton from "../ProfileButton/ProfileButton"
 
 const TestimonyRow = ({ testimony }) => {
   const { result: bill } = useBill(testimony.billId)
-
+  const router = useRouter()
   const profile = usePublicProfile(testimony.authorUid)
   const authorPublic = profile.result?.public
 
   return (
     <tr>
       <td>
-        <Link href={`/bill?id=${testimony.billId}`}>
-          {formatBillId(testimony.billId)}
-        </Link>
+        <Button
```

Comment:
Revert this button -- addressed in https://github.com/codeforboston/advocacy-maps/pull/390

Reply:
done.